Sources/GenericModule/View/ViewController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  Copyright © 2021 Rosberry. All rights reserved.
+//
+
+import UIKit
+
+open class ViewController<ViewModel: GenericModule.ViewModel, ViewInput, Output>: UIViewController, View {
+    public var output: Output
+    public var viewModel: ViewModel
+
+    private var genericModuleViewOutput: ViewOutput? {
+        output as? ViewOutput
+    }
+
+    // MARK: - Lifecycle
+
+    public required init(viewModel: ViewModel, output: Output) {
+        self.viewModel = viewModel
+        self.output = output
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    public required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
```

Comment:
Lets make this inits also `open` to allow their overriding. Or is making them `open` even needed?

Reply:
`required` makes overriding be available automatically